### DISCUSSION TITLE
Version 1.0.8.0

### DISCUSSION THREAD

**Discussion by kmaki565** [kmaki565/EventLook#35] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/kmaki565/EventLook/releases/tag/1.0.8.0'>Version 1.0.8.0</a>.</em>

**Comment by SANCLA** [comment]
Any possibility to download just the executable instead of the code?

**Reply by kmaki565** [reply]
Usually installing from [Microsoft Store](https://www.microsoft.com/store/apps/9NJV5FQ089Z0) would be the best choice, but if you want to try a nightly build:

Go to [Actions](https://github.com/kmaki565/EventLook/actions), click the latest run, find artifact, and right click to "Save link as". Please note artifacts will be deleted after [90 days](https://docs.github.com/en/organizations/managing-organization-settings/configuring-the-retention-period-for-github-actions-artifacts-and-logs-in-your-organization). 
![image](https://github.com/kmaki565/EventLook/assets/16055659/bb35dfd8-a521-4295-abe4-fb0a093b1898)
